src/client/datascience/jupyter/debuggerVariables.ts

```diff
@@ -300,10 +267,6 @@ export class DebuggerVariables extends DebugLocationTracker
 
             this.updateVariables(undefined, message as DebugProtocol.VariablesResponse);
```

Comment:
Can't this stuff be deleted, it seems like you copied the code to line 469

Reply:
I thought it was needed, but after testing, its not, I'll remove it.